### PR TITLE
[fix](cloud-mow) Fix sending commiting rpc to FE twice problem

### PR DESCRIPTION
**Pull request by hust-hhb**
Here is an expample while commit rpc will send twice:
1. first commit request try to get delete bitmap lock, there is 2 lock(fe and ms), which take over rpc timeout(60s default) but not send DELETE_BITMAP_LOCK_ERR to be, and fe will continue to send calculate delete bitmap task to be 
2. be calculate delete bitmap success and remove delete bitmap cache
3. because step 1 take over 60s, be will resend commit rpc to fe
4. after first commit request done, the second commit request from step 3 will do the same thing, but  delete bitmap cache has been delete by first commit, so it will fail on be
5. client will see commit fail

this pr check transaction status before sending delete bitmap task to be, if transaction status is committed or visible, it no need to recalculate delete bitmap again, just retrun rpc success to be.

### CODE REVIEW

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Review by zhannngchen — DISMISSED** [review]
LGTM

**Comment by hust-hhb** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/CloudGlobalTransactionMgr.java [hunk 1]
```diff
@@ -475,6 +475,19 @@ private void commitTransaction(long dbId, List<Table> tableList, long transactio
 
         List<OlapTable> mowTableList = getMowTableList(tableList, tabletCommitInfos);
         if (!mowTableList.isEmpty()) {
+            TransactionState transactionState = Env.getCurrentGlobalTransactionMgr()
```
[the remark this answers is not in the document]
Comment (hust-hhb) [reply fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/CloudGlobalTransactionMgr.java:482]: @zhannngchen done

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by zhannngchen** [comment]
run performance

**Pull request merged**